src/arch/riscv/insts/mem.cc

```diff
@@ -55,8 +55,12 @@ std::string
 Store::generateDisassembly(Addr pc, const loader::SymbolTable *symtab) const
 {
     std::stringstream ss;
-    ss << mnemonic << ' ' << registerName(srcRegIdx(1)) << ", " <<
-        offset << '(' << registerName(srcRegIdx(0)) << ')';
+    if (_numSrcRegs == 1)
+        ss << mnemonic << ' ' << offset << '(' << registerName(srcRegIdx(0))
+           << ")";
+    else
+        ss << mnemonic << ' ' << registerName(srcRegIdx(1)) << ", " <<
+            offset << '(' << registerName(srcRegIdx(0)) << ')';
```

Comment:
```suggestion
    if (_numSrcRegs == 1) {
        ss << mnemonic << ' ' << offset << '(' << registerName(srcRegIdx(0))
           << ")";
    } else {
        ss << mnemonic << ' ' << registerName(srcRegIdx(1)) << ", " <<
            offset << '(' << registerName(srcRegIdx(0)) << ')';
   }
```

Usually it's just a polite request to use `{` around single-line if statements, but this one is required, IMO. Having multiple lines without `{` is *incredibly* likely to generate bugs in the future.

Reply:
Done